Sources/Flatten.swift

```diff
@@ -53,16 +73,21 @@ public enum FlattenStrategy: Equatable {
 	/// The resulting producer will complete when:
 	/// 1. The producer-of-producers and the first "alive" producer has completed.
 	/// 2. The producer-of-producers has completed without inner producer being "alive".
-	case race
+	public static var race: FlattenStrategy {
+		return FlattenStrategy(base: .race)
+	}
 
 	public static func ==(left: FlattenStrategy, right: FlattenStrategy) -> Bool {
-		switch (left, right) {
+		switch (left.base, right.base) {
 		case (.latest, .latest):
 			return true
 
 		case (.concurrent(let leftLimit), .concurrent(let rightLimit)):
 			return leftLimit == rightLimit
 
+		case (.race, .race):
+			return true
+
 		default:
 			return false
```

Comment:
Not saying I agree with this change, but if we merge it, it doesn’t make sense to keep the `Equatable` conformance.